### PR TITLE
Variety of copy edits, Markdown formatting, small fixups for GitHub

### PR DESCRIPTION
**Pull request by jlevy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,31 @@
-# What every software engineer should know about search
+# The Open Guide to Search Engineering
 
-## Want to build or improve a search experience? Start here.
+**What every software engineer should know about search**
 
-Ask a software engineer: “[How would you add search functionality to your product?](https://stackoverflow.com/questions/34314/how-do-i-implement-search-functionality-in-a-website)” or “[How do I build a search engine?](https://www.quora.com/How-to-build-a-search-engine-from-scratch)” You’ll probably immediately hear back something like: “Oh, we’d just launch an ElasticSearch cluster. Search is easy these days.”
+Want to build or improve a search experience? Start here.
+
```
Comment (forwidur) [inline README.md:6]: That's nice! How did you generate the TOC?

File: README.md [hunk 2]
```diff
@@ -138,9 +152,9 @@ Lately, 🔹 [learning to rank](https://en.wikipedia.org/wiki/Learning_to_rank),
 
 A new, [vector space based approach](https://arxiv.org/abs/1708.02702) for semantic retrieval and ranking is gaining popularity lately. The idea is to learn individual low-dimensional vector document representations, then build a model which maps queries into the same vector space.
 
-Then, retrieval is just finding several documents that are closest by some metric (e.g. Eucledian distance) to the query vector. Ranking is the distance itself. If the mapping of both the documents and queries is built well, the documents are chosen not by a fact of presence of some simple pattern (like a word), but how close the documents are to the query by *meaning*.
+Then, retrieval is just finding several documents that are closest by some metric (e.g. Euclidian distance) to the query vector. Ranking is the distance itself. If the mapping of both the documents and queries is built well, the documents are chosen not by a fact of presence of some simple pattern (like a word), but how close the documents are to the query by *meaning*.
```
Comment (forwidur) [inline README.md]: E: It's `Euclidean`: https://en.wikipedia.org/wiki/Euclidean_distance. ;-)

File: README.md [hunk 3]
```diff
@@ -279,6 +293,16 @@ This concludes my humble attempt to make a somewhat-useful “map” for an aspi
...
-Special thanks to* *[Joshua Levy](https://twitter.com/ojoshe),* *[Leo Polovets](https://twitter.com/lpolovets)* *and* *[Abhishek Das](https://www.linkedin.com/in/abhishek-das-3280053/)* *for reading drafts of this and their invaluable feedback!
+## Credits
+
+Original Author: [**Max Grigorev**](https://twitter.com/forwidur)
+
+Editorial help and feedback:
+[**Joshua Levy**](https://twitter.com/ojoshe),
+[**Leo Polovets**](https://twitter.com/lpolovets), and [**Abhishek Das**](https://www.linkedin.com/in/abhishek-das-3280053/)
+
+## License
```
Comment (forwidur) [inline README.md:304]: D: This could go into a separate LICENSE file. Not a strong opinion, but that seems to be the pattern usually.

**Review by forwidur — APPROVED** [review]
LGTM after the error gets fixed. Awesome job!

File: README.md (hunk 1 above)
Comment (jlevy) [reply README.md:6]: Using https://github.com/jonschlinkert/markdown-toc

File: README.md (hunk 2 above)
Comment (jlevy) [reply README.md]: Fixed.

**Comment by jlevy** [comment]
👍 Great. Typo fixed. Thanks!

**Pull request merged**